src/snake/core.clj

```diff
@@ -227,7 +226,7 @@
           :when                             (pos? len)
           :let                              [dest (str local-path "/" path)]]
     (io/make-parents dest)
-    (with-open [raw-stream (:content (get-object bucket path))
+    (with-open [raw-stream (:content (get-object bucket path)) ;; TODO
```

Comment:
@chris-etheridge I'm quite confused here and didn't have any way to test.

`get-object` uses slurp, so technically it shoudl return a string – so I don't understand how you can be calling `:content` on it – I must be missing something?